libs/explorers/models/src/lib/comparison-tool.ts

```diff
@@ -26,6 +26,7 @@ export interface ComparisonToolViewConfig {
   visualizationOverviewPanes: VisualizationOverviewPane[];
   rowsPerPage: number;
   rowIdDataKey: string;
+  defaultSort?: Array<{ field: string; order: 1 | -1 }>;
```

Comment:
We can use `defaultSort` to specify custom default sorts when navigating to a CT